content/exchange/artifacts/Exchange.Windows.EventLogs.Hayabusa.Takajo

```diff
@@ -0,0 +1,94 @@
+name: Exchange.Windows.EventLogs.Hayabusa.Takajo
+description: |
+   [Hayabusa](https://github.com/Yamato-Security/hayabusa) is a
+   Windows event log fast forensics timeline generator and threat
+   hunting tool.
+
+   This artifact runs Hayabusa on the endpoint against the specified
+   Windows event log directory, and generates and uploads a single CSV/JSONL
+   file for further analysis with excel, timeline explorer, elastic
+   stack, jq, etc.
+
+   [Takajo] (https://github.com/Yamato-Security/takajo) is a fast forensics analyzer
+   for Hayabusa results written in Nim. Takajō means "Falconer" in Japanese
+   and was chosen as it analyzes Hayabusa's "catches" (results).
+
+author: Eric Capuano - @eric_capuano, Whitney Champion - @shortxstack, Zach Mathis - @yamatosecurity
+
+tools:
+ - name: Hayabusa-2.14.0
+   url: https://github.com/Yamato-Security/hayabusa/releases/download/v2.14.0/hayabusa-2.14.0-win-x64.zip
```

Comment:
would it be better to just call the other artifact? It will be better maintained?

Reply:
Yes, I didn't notice about that.

Reply:
You dont seem to be calling the existing one but you removed this tool - this will break this artifact

Reply:
My fault! Maybe I dont understand about what do you mean to call the existing one.
Thank you

Reply:
I don't think its a good idea to call the existing one because it uses different parameters that I don't, I should remove and replace for those I actually calling.
If Im wrong I would apreciate to improve the code.

Reply:
Ok thats fine - if you use Hayabusa in a different way then its ok to call it different. Just fix the presubmit (file does not have a .yaml extention)

Reply:
Done